enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx

```diff
@@ -98,89 +36,87 @@ export const MetabaseProviderInternal = ({
   pluginsConfig,
   eventHandlers,
   theme,
-  store,
-  className,
+  reduxStore,
   locale,
   errorComponent,
   loaderComponent,
   allowConsoleLog,
 }: InternalMetabaseProviderProps): JSX.Element => {
   const { fontFamily } = theme ?? {};
-  useInitData({ authConfig, allowConsoleLog });
+
+  // The main call of useInitData happens in the root MetabaseProvider
+  // This call in the component-level MetabaseProvider is still needed for:
+  // - Storybook stories, where we don't have the root MetabaseProvider
+  // - Unit tests
+  useInitData({ reduxStore, authConfig, allowConsoleLog });
```

Comment:
I wanted to remove it from here and add on the level of storybook/unit tests.
But after all it requires too many places to add it instead of just having it here